MyMentorProject/src/main/java/com/Deny/MyMentorProject/Services/UserService.java

```diff
@@ -0,0 +1,51 @@
+package com.Deny.MyMentorProject.Services;
+
+import com.Deny.MyMentorProject.User.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    @Autowired
+    User user;
+
+
+  public UserService(User user) {       this.user = user;
+ }
+
+
+    public String helloMessage() {
+        return "Hello My Friend !";}
+
+    public String helloUser() {
+        user.setName("Joe");
+        return "hello  " + user.getName();
+    }
+    public String messageToReTurn(String messagerequest) {
+
+        String reply = "";
+
+        if (messagerequest ==""){
```

Comment:
Feature 3 should be a random greeting. You can create a list of the statements below and call a randomIndex using java.util.Random.

Hello {User}!
Hey {User}, nice to see you here!
{User} welcome back!
Have a splendid day {User}.